src/gtfs/mod.rs

```diff
@@ -18,10 +18,24 @@ mod read;
 
 use std::path;
 use {Collections, PtObjects};
+use std::fs::File;
+use gtfs::read::Config;
 
-pub fn read<P: AsRef<path::Path>>(path: P) -> PtObjects {
-    let path = path.as_ref();
+extern crate serde_json;
+
+pub fn read<P: AsRef<path::Path>>(path: P, config_path: Option<P>) -> PtObjects {
```

Comment:
we should instead now return a `Result<PtObjects>` (see https://github.com/CanalTP/navitia_model/blob/master/src/lib.rs#L48)